components/widgets/icons/camera_icon.jsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+export default class CameraIcon extends React.PureComponent {
+    render() {
+        return (
+            <span {...this.props}>
+                <FormattedMessage
+                    id='generic_icons.camera'
+                    defaultMessage='Camera Icon'
+                >
+                    {(ariaLabel) => (
```

Comment:
Technically, this is incorrect typing since `ariaLabel` might not be a string at this point depending on the message being used here. If you convert it to a functional component though, you can use the `useIntl` hook to ensure you get a string here though. It's like `injectIntl`, but simpler.

```typescript
import {useIntl} from 'react-intl';

export default function CameraIcon(props: Props) {
    const intl = useIntl();

    return (
        <span {...props}>
            <svg aria-label={intl.formatMessage({id: 'generic_icons.camera', defaultMessage: 'Camera Icon'})}></svg>
        </span>
    );
}
```